text/0036-ternary.md

```diff
@@ -0,0 +1,254 @@
++ Feature name: ternary
++ Start date: 2020-05-28
++ RFC PR: [iotaledger/bee-rfcs#36](https://github.com/iotaledger/bee-rfcs/pull/36)
++ Bee issue: [iotaledger/bee#80](https://github.com/iotaledger/bee/issues/80)
+
+# Summary
+
+This RFC introduces both the API and implementation of the `bee-ternary`
+crate, a general-purpose ternary manipulation crate written in Rust that may be
+used to handle ternary data in the IOTA ecosystem.
+
+# Motivation
+
+Ternary has been a fundamental part of the IOTA technology and is used
+throughout, even for fundamental features like wallet addresses and transaction
```

Comment:
Is it just me or this `even` sounds a bit negative ?

Reply:
No, doesn't sound negative. The point Josh is making here is that ternary is central to IOTA tech down to its core functionalities.

Reply:
But I would rephrase this sentence to the one I suggested :stuck_out_tongue: